tests/integration/examples_test.py

```diff
@@ -163,3 +165,18 @@ def test_port_vlan(eth1_up):
         assertlib.assert_state(desired_state)
 
     assertlib.assert_absent("linux-br0")
+
+
+@pytest.mark.xfail(
+    is_nm_lower_or_equal_than_1_22_16(),
+    reason="Using the interface name is supported only with NM 1.22.16 or "
```

Comment:
Is this really also fixed in 1.24.0?

Reply:
No it does not. But our CI does not test 1.24 branch.

Reply:
Done.